contrib/librdkafka-cmake/config.h.in

```diff
@@ -66,7 +66,7 @@
 #cmakedefine WITH_SASL_OAUTHBEARER 1
 #cmakedefine WITH_SASL_CYRUS 1
 // crc32chw
-#if !defined(__PPC__) && !defined(__riscv) && !defined(__aarch64__) && !defined(__s390x__)
+#if !defined(__PPC__) && !defined(__riscv) && !defined(__aarch64__) && !defined(__s390x__) && !defined(__loongarch64)
```

Comment:
Related to the changes in contrib/boringssl/.

Please don't submit this until https://github.com/ClickHouse/ClickHouse/pull/59870 is merged. The other PR takes precedence for now.